src/DIRAC/Core/Security/TokenFile.py

```diff
@@ -0,0 +1,99 @@
+""" Collection of utilities for dealing with security files (i.e. token files)
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import os
+import json
+import stat
+import tempfile
+
+from DIRAC import S_OK, S_ERROR
+from DIRAC.Core.Utilities import DErrno
+from DIRAC.Core.Security.Locations import getTokenLocation
+
+
+def readTokenFromFile(fileName=None):
+  """ Read token from a file
+
+      :param str fileName: filename to read
+
+      :return: S_OK(dict)/S_ERROR()
+  """
+  if not fileName:
+    fileName = getTokenLocation() or os.environ.get('DIRAC_TOKEN_FILE', "/tmp/JWTup_u%d" % os.getuid())
+  try:
+    with open(fileName, 'r') as f:
```

Comment:
```suggestion
    with open(fileName, 'rt') as f:
```

as per https://dirac.readthedocs.io/en/latest/DeveloperGuide/Python3Migration/index.html

Reply:
The default is text mode. I like to be explicit but I'm in a minority for the Python community as a whole.